File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        for ele in numbers:
            indexx = target-ele 
            try:
                otherIndex = numbers.index(indexx)
    
                return [numbers.index(ele)+1,otherIndex+1]
            except:
                continue
            

File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        // <otherValue,index>
        HashMap<Integer,Integer> seen = new  HashMap<>();
        for (int i = 0;i<n;i++){
            int OtherValue = target - nums[i];
            int index = seen.getOrDefault(OtherValue,-1);
            if (index!=-1){
                return new int[]{index,i};
            }
            seen.put(nums[i],i);
        }
        return new int[]{0,0};
    }
}
